### ISSUE TITLE
Issue with running the tool

### ISSUE THREAD

**Issue by puriaayush01** [OWASP/OFFAT#140]
Hi @dmdhrumilmistry / @OWASPFoundation / @nrathaus ,

**1) I wanted to know if the OFFAT tool can run for the VAmPI API?**

You can clone the project from [https://github.com/erev0s/VAmPI] and run app.py to start the server. 
The JSON file can be found at this URL: http://127.0.0.1:5000/openapi.json

**2) In case I have a bearer token or the client id & secret (which is used to generate the token)- to authenticate my API, what changes do I need to make in the case of OpenAPI v3?** 

Could you guide me if that works for you?

Thanks!

**Comment by dmdhrumilmistry** [comment]
Server URL is missing in JSON file.

![image](https://github.com/user-attachments/assets/6dffba06-324d-4c3e-8fab-5375e80745a8)

Fix:

![image](https://github.com/user-attachments/assets/f65d0c85-61b6-4932-97c0-a36dea791fb1)

![image](https://github.com/user-attachments/assets/b3894c58-e7e9-420e-973f-3aa70258699e)

**Comment by puriaayush01** [comment]
Even after making the change, it still doesn't work for me like it did for you. Getting some Parser Error. 
Which branch are you using? I'm currently on the master branch - could that be the reason? 

Also, did you download the JSON file and run the query using "**offat -f filename.json**"?

**Comment by puriaayush01** [comment]
It works for me; thank you @dmdhrumilmistry !

Regarding the Header Flag, I attempted to run an API documentation that requires an access token for authorization, so I passed the bearer token along with the header:

**offat -f swagger_file.json -H 'Accept: application/json' -H 'Authorization: Bearer YourJWTToken'**

However, I'm facing some issues. Would it be possible for us to connect over a Teams call or Google Meet? This way, I can clearly explain what I’m trying to achieve. Without the authorization, the OFFAT tool is unable to test each end point.
